src/components/ProductForList.js

```diff
@@ -26,17 +31,20 @@ export const ProductForList = ({ item, handleDeleteAttempt, token }) => {
 
   useEffect(() => {
     setIsBought(validateHours(item, 24));
+    setIsInactive(validateActive(item));
   }, [item]);
 
   return (
     <div
       className="product-container"
       aria-label={`${
-        item.howSoon === 7
-          ? 'soon'
-          : item.howSoon === 14
+        isInactive
+          ? 'inactive'
+          : item.howSoon > 30
+          ? 'not soon'
+          : item.howSoon > 6
```

Comment:
Considero, que ya es un buen momento para crear una función (y mejor si se lo extrae a otro archivo como el `utils.js`)

```javascript
  const productAriaLabel = (isInactive = false, howSoon = 0) => {
    if (isInactive) return "inactive"

    if (howSoon < 7) {
      return "soon"
    } else if (howSoon >= 7 && howSoon <= 30) {
      return "kind of soon"
    } else if (howSoon > 30) {
      return "not soon"
    }
  }
```

```suggestion
<div className="product-container" aria-label={productAriaLabel(isInactive, item.howSoon)}>
```

Reply:
De acuerdo, está hecho.